cdap-ui/app/cdap/components/Lab/ExperimentalFeature/index.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import * as React from 'react';
+import If from 'components/If';
+
+interface IExperimentWrapperProps {
+  children: React.ReactElement<any>;
```

Comment:
nit: `React.ReactNode`

Reply:
Had to make changes in <If> component as well. Had to wrap `children` with React Fragment. More info here: https://github.com/DefinitelyTyped/DefinitelyTyped/issues/18051